src/OrchardCore.Modules/OrchardCore.Demo/Startup.cs

```diff
@@ -77,7 +77,7 @@ public override void ConfigureServices(IServiceCollection services)
                 options.Conventions.AddModularPageRoute("/OrchardCore.Demo/Pages/Hello", "Hello");
 
                 // This declaration would define an home page
-                options.Conventions.AddModularPageRoute("/OrchardCore.Demo/Pages/Hello", "");
+                // options.Conventions.AddModularPageRoute("/OrchardCore.Demo/Pages/Hello", "");
```

Comment:
explain the comment

Reply:
This line maps `/OrchardCore.Demo/Pages/Hello` to "" so it becomes the home page.
It's just an example that i forgot to re-comment when i was working on page routes.

Then, because here i started my tests with the `OC.Demo`, i referenced again the demo module, then i saw that the home page was the hello page.